### DISCUSSION TITLE
Notification when a password has reached its expiration date

### DISCUSSION THREAD

**Discussion by EliasColl** [keepassxreboot/keepassxc#11356] [Ideas]
I would like to see a notification in Windows when one of my passwords expires so that I can proceed to change it. I would also like to see some buttons in case I want to postpone that change and another button to open the web to log in and proceed to change it quickly. I know that it could be an important feature and easy to implement.

**Comment by droidmonkey** [comment]
We have already implemented that on database unlock, it's enabled by default. We wouldn't do that with notifications.
